public/locales/ar/translation.json

```diff
@@ -224,7 +224,16 @@
             "type": "النوع"
         },
         "title": "التنزيلات",
-        "ETA": "الوقت المقدَّر"
+        "ETA": "الوقت المقدَّر",
```

Comment:
translations shouldn't be part of the repo, we manage translations in weblate and it takes care of updating the repo later https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki/Translations#how-to-translate-heroic